docs/source/conf.py

```diff
@@ -326,14 +326,11 @@
 # Customized html_title here.
 # Default is " ".join(project, release, "documentation") if not set
 if RELEASE:
-    # remove hash (start with 'a') from version number if any
-    version_end = torch_version.find('a')
-    if version_end == -1:
```

Comment:
What was the case where this would not find the hash?  Is it guaranteed that the version here is well formed as `X.Y...` ?

Reply:
Ok! Thanks for the details.